vmdb/app/assets/javascripts/miq_application.js

```diff
@@ -1002,7 +1007,7 @@ function miqCheckMaxLength(obj) {
   if (obj.getAttribute && obj.value.length > ml) {
     obj.value = obj.value.substring(0, ml);
   }
-  if (counter != "") {
+  if (counter) {
```

Comment:
@martinpovolny I have tested this case, and it seems it behaves same way

    var abc = document.getElementById('fullscreen_overlay');
    var bbz = abc.getAttribute('bzz');

    bzz == ""
    => false
    Boolean(bzz)
    => false

    bzz != ""
    => true
    Boolean(!bzz)
    => true